src/containers/Errors/index.tsx

```diff
@@ -1,12 +1,44 @@
+import * as Sentry from '@sentry/browser';
 import * as React from 'react';
 
-class ErrorWrapper extends React.Component {
+interface ErrorWrapperState {
+    eventId: any; // tslint:disable-line
```

Comment:
Define type here instead of using ```any```

Reply:
EventId can be presented in different types